sources/packages/web/src/views/aest/student/StudentApplicationAppeals.vue

```diff
@@ -45,7 +45,13 @@
               </span>
             </template>
           </Column>
-          <Column field="fullName" header="Name" :sortable="true"> </Column>
+          <Column field="firstName" header="Given names">
+            <template #body="slotProps">
+              <span>{{ slotProps.data.firstName || "" }}</span>
```

Comment:
We have formatter called `emptyStringFiller` please use that. 
![image](https://github.com/user-attachments/assets/f61d8af9-c7cc-4523-84cb-a95b74f40016)
![image](https://github.com/user-attachments/assets/36e5c3cc-e40c-4c98-bffe-14853abda964)